frontend/src/styles.css

```diff
@@ -5,11 +5,7 @@
   font-weight: 300;
   src: url('assets/fonts/roboto-v18-latin-300.eot'); /* IE9 Compat Modes */
   src: local('Roboto Light'), local('Roboto-Light'),
-  url('assets/fonts/roboto-v18-latin-300.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
-  url('assets/fonts/roboto-v18-latin-300.woff2') format('woff2'), /* Super Modern Browsers */
-  url('assets/fonts/roboto-v18-latin-300.woff') format('woff'), /* Modern Browsers */
-  url('assets/fonts/roboto-v18-latin-300.ttf') format('truetype'), /* Safari, Android, iOS */
-  url('assets/fonts/roboto-v18-latin-300.svg#Roboto') format('svg'); /* Legacy iOS */
+  url('assets/fonts/roboto-v18-latin-300.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */ url('assets/fonts/roboto-v18-latin-300.woff2') format('woff2'), /* Super Modern Browsers */ url('assets/fonts/roboto-v18-latin-300.woff') format('woff'), /* Modern Browsers */ url('assets/fonts/roboto-v18-latin-300.ttf') format('truetype'), /* Safari, Android, iOS */ url('assets/fonts/roboto-v18-latin-300.svg#Roboto') format('svg'); /* Legacy iOS */
```

Comment:
Did this get reformatted automatically?

Reply:
@palango probably yes let me go and fix it